vrecord

```diff
@@ -798,18 +798,20 @@ shift $(( ${OPTIND} - 1 ))
 
 if [[ "${runtype}" = "GUI" ]] ; then
     _master_gui
-elif [[ "${runtype}" = "reset" ]] ; then
+fi
+if [[ "${runtype}" = "reset" ]] ; then
     _report -q -n "Reseting the configuration will clear ${config_file}. Please enter [Y] to confirm: "
     read reset_response
-    if [[ "${reset_response}" = [Yy] ]] ; then
+    if [[ "${reset_response}" = "[Y]" ]] ; then
         _report -d "Clearing ${config_file}."
         echo -n "" > "${config_file}"
         runtype="edit"
     else
         _report -d "Reset aborted. Exiting."
         exit 0
     fi
-elif [[ "${runtype}" = "edit" ]] ; then
+fi
+if [[ "${runtype}" = "edit" ]] ; then
```

Comment:
I think this all could have been kept as a `if elif fi` rather than changing it back into two `if fi` `if fi` blocks

Reply:
when i tested it that way, it said that there was a disagreement